en/setup/dev_env_linux.md

```diff
@@ -14,63 +37,113 @@ sudo usermod -a -G dialout $USER
-* Host simulation: [jMAVSim SITL](../simulation/sitl.md) and [Gazebo SITL](../simulation/gazebo.md)
+Ubuntu comes with a serial modem manager which interferes heavily with any robotics related use of a serial port \(or USB serial\). It can removed/deinstalled without side effects:
+
+```sh
+sudo apt-get remove modemmanager
+```
+
+## Ninja Build System
 
-> **Info** Install the [Ninja Build System](../setup/dev_env_linux_boutique.md#ninja-build-system) for faster build times than with Make. It will be automatically selected if installed.
+Ninja is fast than Make and the PX4 CMake generators support it. Unfortunately Ubuntu carries only a very outdated version at this point. To install a recent version of [Ninja](https://github.com/martine/ninja), download the binary and add it to your path:
+
+<div class="host-code"></div>
+
+```sh
+mkdir -p $HOME/ninja
+cd $HOME/ninja
+wget https://github.com/martine/ninja/releases/download/v1.6.0/ninja-linux.zip
+unzip ninja-linux.zip
+rm ninja-linux.zip
+exportline="export PATH=$HOME/ninja:\$PATH"
+if grep -Fxq "$exportline" ~/.profile; then echo nothing to do ; else echo $exportline >> ~/.profile; fi
+. ~/.profile
+```
+
+## Common Dependencies
+
+Update the package list and install the following dependencies for all PX4 build targets. 
+
+> **Info** Install the [Ninja Build System](#ninja-build-system) for faster build times than with `Make`. It will be automatically selected if installed.
 
 ```sh
 sudo add-apt-repository ppa:george-edison55/cmake-3.x -y
 sudo apt-get update
 sudo apt-get install python-argparse git-core wget zip \
     python-empy qtcreator cmake build-essential genromfs -y
-# simulation tools
-sudo apt-get install ant protobuf-compiler libeigen3-dev libopencv-dev openjdk-8-jdk openjdk-8-jre clang-3.5 lldb-3.5 -y
 # required python packages
 sudo apt-get install python-pip
 sudo -H pip install pandas jinja2
 ```
 
-### NuttX based hardware
+## Simulation Dependencies
+The dependencies for the Gazebo and jMAVSim simulators listed below. You should minimally install jMAVSim to make it easy to test the installation. Additional information about these and other supported simulators is covered in: [Simulation](../simulation/README.md).
 
-Ubuntu comes with a serial modem manager which interferes heavily with any robotics related use of a serial port \(or USB serial\). It can deinstalled without side effects:
+### jMAVSim
 
-```sh
-sudo apt-get remove modemmanager
+Install the dependencies for [jMAVSim Simulation](../simulation/sitl.md).
+
+```
+# jMAVSim simulator
+sudo apt-get install ant openjdk-8-jdk openjdk-8-jre -y
 ```
 
-Update the package list and install the following dependencies. Packages with specified versions should be installed with this particular package version.
+### Gazebo
+
+Install the dependencies for [Gazebo Simulation](../simulation/gazebo.md).
+
+```
+# Gazebo simulator
+sudo apt-get install protobuf-compiler libeigen3-dev libopencv-dev -y
```

Comment:
@LorenzMeier @ChristophTobler @vilhjalmur89 
OK, to address this I have:
1. Added a [new section](https://github.com/hamishwillee/Devguide/blob/tidy_toolchain/en/setup/dev_env_linux.md#rosgazebo) for ROS/Gazebo installation. This includes links to where I got the information.
2. Added a [new build script](https://raw.githubusercontent.com/hamishwillee/Devguide/tidy_toolchain/build_scripts/ubuntu_sim_ros_gazebo.sh) following those instructions (linked from [here](https://github.com/hamishwillee/Devguide/blob/tidy_toolchain/en/setup/dev_env_linux.md#scripts)).

The script installs the default ROS Kinect (with gazebo7) and MAVROS from source (as recommended in our docs).

@ChristophTobler Can you please sanity check that the script is sane/reasonable best approach? 

I have tested the script using approach suggested by Christoph:
1. Open terminal (1)
```
export PX4_HOME_LON=8.451433399999985
export PX4_HOME_LAT=47.4028213
make posix_sitl_default gazebo
```
This just tells the simulation where it should start globally. 

2. Open second terminal and do
```
roslaunch mavros px4.launch fcu_url:="udp://:14540@127.0.0.1:14557"
```
2. Open  third terminaland do
```
rosrun mavros mavsafety arm
rosrun mavros mavcmd takeoff 0.0 0.0 47.4028213 8.451433399999985 3.0
rosrun mavros mavcmd land 0.0 47.4028213 8.451433399999985 0.0
```